roles/etcd_ca/tasks/main.yml

```diff
@@ -38,9 +38,9 @@
   delegate_to: "{{ etcd_ca_host }}"
   run_once: true
 
-- command: touch {{ etcd_ca_db }}
-  args:
-    creates: "{{ etcd_ca_db }}"
+- file:
+    path: "{{ etcd_ca_db }}"
+    state: touch
```

Comment:
`run_once` will cause it to be ran just once in a single run (many hosts delegating to a single host and only running on the single `etcd_ca_host`) but subsequent runs of openshift-ansible will touch the file. If we want to remove the warning we may have to check if the file exists or come up with an alternative.